packages/keystatic/src/app/shell/common.tsx

```diff
@@ -1,6 +1,36 @@
-import { breakpointQueries, css, tokenSchema } from '@keystar/ui/style';
+import { useProvider } from '@keystar/ui/core';
+import {
+  breakpointQueries,
+  css,
+  tokenSchema,
+  useMediaQuery,
+} from '@keystar/ui/style';
 
-export function ZapLogo() {
+import { useConfig } from './context';
+
+export function useBrand() {
+  let { colorScheme } = useProvider();
+  let config = useConfig();
+  let prefersDark = useMediaQuery('(prefers-color-scheme: dark)');
+
+  let brandMark = <ZapLogo />;
+  let brandName = config.cloud ? config.cloud.project : 'Keystatic';
```

Comment:
This means we're not showing the GitHub repo when in GitHub mode?

Reply:
TBH I've always found it strange that was so prominent; very easy to misunderstand and click assuming it will take the user "home". The repo is still available from the git menu:

<img width="150" alt="image" src="https://github.com/Thinkmill/keystatic/assets/2730833/6cf2a4e9-d69f-4f13-ad87-c06dc6e5917c">

Reply:
Sure, I don't mind about it not being a link but it seems a bit weird to show the cloud project key when in cloud mode but not the repo in github mode?

Reply:
Oh right, you mean default it? Yeah, for sure